node/src/accountant/scanners/mod.rs

```diff
@@ -383,60 +382,52 @@ impl PayableScanner {
             .iter()
             .map(|pending_payable| pending_payable.hash)
             .collect::<Vec<H256>>();
-        let sent_payables_hashmap = sent_payables
+        let mut sent_payables_hashmap = sent_payables
             .iter()
             .map(|payable| (payable.hash, &payable.recipient_wallet))
             .collect::<HashMap<H256, &Wallet>>();
 
         let transaction_hashes = self.pending_payable_dao.fingerprints_rowids(&hashes);
-        let mut rowid_pairs_with_hashes = transaction_hashes
+        let mut hashes_from_fb_comp = transaction_hashes
             .rowid_results
             .iter()
             .map(|(_rowid, hash)| *hash)
             .collect::<HashSet<H256>>();
         for hash in &transaction_hashes.no_rowid_results {
-            rowid_pairs_with_hashes.insert(*hash);
+            hashes_from_fb_comp.insert(*hash);
         }
         let sent_payables_hashes = hashes.iter().copied().collect::<HashSet<H256>>();
 
-        if !PayableScanner::is_symmetrical(sent_payables_hashes, rowid_pairs_with_hashes) {
+        if !PayableScanner::is_symmetrical(sent_payables_hashes, hashes_from_fb_comp.clone()) {
```

Comment:
The clone may not be what you want to do. Removed? Otherwise consider providing the function `is_symetrical` with references to the HashSets.